_posts/2016-11-18-JAX-London-2016.md

```diff
@@ -0,0 +1,691 @@
+comments: true
+---
+> JAX London is a four-day conference for cutting-edge software engineers and enterprise-level professionals. JAX brings together the world’s leading innovators in the fields of Java, microservices, Continuous Delivery and DevOps.
+> For this year's slogan they decided on: "Create, Innovate, Code".
+> Ordina was present at JAX London on the 11th and 12th of October 2016 where one of our colleagues, Bart Blommaerts, also presented a talk on The Serverless Cloud.
+> In this blogpost we want to give the highlights of some of the talks be followed.
+
+----------
+
+# Day 1
+
+## Introduction to JAX London - Sebastian Meyen
+
+<img class="float-image p-image" src="{{ '/img/jax-london-2016/sebastian-meyen.jpeg' | prepend: site.baseurl }}" alt="Sebastian Meyen" style="width: 100px"/>
+
+To start off, [Sebastian Meyen](https://twitter.com/smeyen), program chair of the JAX conferences, gave an introduction to JAX London.
+He mentioned that only half of the attendants of JAX originate from the UK, making it an international conference.
+JAX is all about "openess", we should be celebrating open source and embrace open source thinking as it makes our code smarter.
+Java is one of the most powerful ecosystems out there according to him and it comes with a unique culture.
+
+Sebastian explained that there are different cultures within a company:
+
+* Pioneers: They go into the wild and experiment and although they might fail often, they're looking at what the next thing might be
+* Settlers: They make a valid business model based on the results of the research of the pioneers and make stable technology for it
+* Town builders: They look at the portfolio of the settlers and decide what to industrialise, they want to create volume
+
+He stresses that innovation happens on all three levels and not just only at the pioneers or settlers.
+Furthermore he went briefly over the different genres of the talk and the conference app before introducing James Governor.
+
+## Opening keynote: Java for Cloud Natives - James Governor
+
+<img class="float-image p-image" src="{{ '/img/jax-london-2016/james-governor.jpg' | prepend: site.baseurl }}" alt="James Governor" style="width: 100px"/>
+
+[James Governor](https://twitter.com/monkchips) opens up his talk by showing how Java is still high on the Programming Language rankings despite the "Java is dead"-doomspeak every now and then.
+Regarding Java frameworks, Spring and Spring Boot are really crushing it.
+He also mentions that most startups usually start with a new and fancy language but as they mature, a lot of them actually turn into Java shops.
+Examples of this being: Uber, LinkedIn, Netflix, Twitter, Amazon, Etsy, Facebook, Yahoo and Google.
+
+Twitter for example started in Ruby and during the US presidential election in 2012 they migrated to Scala and Java on the JVM for scalability and performance reasons.
+After the migration they managed to sustain peaks of TPS (Tweets Per Second) for hours.
```

Comment:
Added a link to https://blog.twitter.com/2012/election-night-2012 (although that's in TPM but they should get the idea).